debug/org.eclipse.ui.console/src/org/eclipse/ui/internal/console/ConsoleMessages.properties

```diff
@@ -58,4 +58,7 @@ TextConsolePage_PasteDescrip=Paste
 PatternMatchListenerExtension_3=Console Pattern Match Listener
 PatternMatchListenerExtension_4=contributed by
 PatternMatchListenerExtension_5=is missing required enablement expression and will be removed
-UpdatingConsoleState=Updating console state
\ No newline at end of file
+UpdatingConsoleState=Updating console state
+TurnOnAutoPinDialogMessage=If a new Console view is opened, it's possible to auto pin the old Console view and new Console view to keep its content. Should we pin the old and new Console views (so different console pages could be updated independently)?
```

Comment:
`To see updates of different console pages in parallel, both current and new Console view will be pinned now.`

Note, this message change implies change from yes/no to OK/Cancel.